jujugui/static/gui/src/app/components/spinner/_spinner.scss

```diff
@@ -1,21 +1,72 @@
 @keyframes spinner {
   0% {
-    @include rotate(0deg);
+    transform: rotate(0deg);
   }
   100% {
-    @include rotate(360deg);
+    transform: rotate(360deg);
```

Comment:
Including the `rotate` mixin gave us the vendor prefixes; is it safe to drop that for a straight transform?